package/utils/helpers.js

```
@@ -25,16 +25,84 @@
 
+const validateBabelDependencies = () => {
+  // Only validate core dependencies that are absolutely required
+  // Additional packages like presets are optional and project-specific
+  const coreRequiredPackages = ["@babel/core", "babel-loader"]
+
+  const missingCorePackages = []
+  for (const pkg of coreRequiredPackages) {
+    if (!moduleExists(pkg)) {
+      missingCorePackages.push(pkg)
+    }
+  }
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)